src/foraging_gui/Foraging.py

```diff
@@ -985,6 +1008,7 @@ def _GetSettings(self):
             'metadata_dialog_folder':os.path.join(self.SettingFolder,"metadata_dialog")+'\\',
             'rig_metadata_folder':os.path.join(self.SettingFolder,"rig_metadata")+'\\',
             'project_info_file':os.path.join(self.SettingFolder,"Project Name and Funding Source v2.csv"),
+            'schedule_path':os.path.join('Z:\\','dynamic_foraging','DynamicForagingSchedule.csv'),
```

Comment:
We should also enable setting 'schedule_path' from the ForagingSettings.json. The path may be different in different computers.

Reply:
The value can be modified by adding an entry in `ForagingSettings.json`. This code just sets the default value. This value is only used as `self.Settings['schedule_path']`

Reply:
That makes sense.